pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -34,9 +35,65 @@ const (
 	testObservationDomainID = 0xabcd
 )
 
-// TODO: We will add another test for sendDataRecord when we support adding multiple records to single set.
 // Currently, we are supporting adding only one record from one flow key to the set.
 
+func TestFlowAggregator_sendFlowKeyRecord(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockIPFIXExpProc := ipfixtest.NewMockIPFIXExportingProcess(ctrl)
+	mockIPFIXRegistry := ipfixtest.NewMockIPFIXRegistry(ctrl)
+	mockDataSet := ipfixtest.NewMockIPFIXSet(ctrl)
+	mockRecord := ipfixtest.NewMockIPFIXRecord(ctrl)
+	mockAggregationProcess := ipfixtest.NewMockIPFIXAggregationProcess(ctrl)
+
+	fa := &flowAggregator{
+		"",
+		"",
+		"tcp",
+		nil,
+		mockAggregationProcess,
+		testExportInterval,
+		mockIPFIXExpProc,
+		testTemplateIDv4,
+		testTemplateIDv6,
+		mockIPFIXRegistry,
+		mockDataSet,
+		"",
+		nil,
+		testObservationDomainID,
+	}
+
+	key1 := ipfixintermediate.FlowKey{"10.0.0.1", "10.0.0.2", 6, 1234, 5678}
```

Comment:
key1 -> v4Key or ipv4Key
key2 -> v6Key or ipv6Key

Reply:
I put it into struct, with a bool showing ipv4/ipv6. Hope it can improve the clarity.
`	key_testcases := []struct {
		key    ipfixintermediate.FlowKey
		isIPv6 bool
	}{
		{
			ipfixintermediate.FlowKey{
				SourceAddress:      "10.0.0.1",
				DestinationAddress: "10.0.0.2",
				Protocol:           6,
				SourcePort:         1234,
				DestinationPort:    5678,
			},
			false,
		},`